compose/snippets/src/main/java/com/example/compose/snippets/layouts/AdaptiveLayoutSnippets.kt

```diff
@@ -51,22 +48,14 @@ import androidx.compose.ui.unit.dp
 * limitations under the License.
 */
 // [START android_compose_adaptive_layouts_basic]
-class MainActivity : ComponentActivity() {
-    @OptIn(ExperimentalMaterial3WindowSizeClassApi::class)
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-
-        setContent {
-            val windowSizeClass = calculateWindowSizeClass(this)
-            MyApp(windowSizeClass)
-        }
-    }
-}
+@OptIn(ExperimentalMaterial3AdaptiveApi::class)
 @Composable
-fun MyApp(windowSizeClass: WindowSizeClass) {
+fun MyApp(
+    windowSizeClass: WindowSizeClass = currentWindowAdaptiveInfo().windowSizeClass
+) {
     // Perform logic on the size class to decide whether to show
```

Comment:
I think the comment could be on one line. Would help readability. See staged page, Window size classes.

Reply:
Updated